crates/ra_parser/src/grammar/params.rs

```diff
@@ -41,9 +44,16 @@ fn list_(p: &mut Parser, flavor: Flavor) {
     let m = p.start();
     p.bump();
     if flavor.type_required() {
+        attributes::outer_attributes(p);
         opt_self_param(p);
     }
     while !p.at(EOF) && !p.at(ket) && !(flavor.type_required() && p.at(T![...])) {
+        attributes::outer_attributes(p);
```

Comment:
...and let' *one* test  above this line.

All the other attribute test in this PR are better put into a `test_data/parser/ok/parameter_attrs.rs`

Reply:
How I should call those tests?